js/modules/k6/http/http.go

```diff
@@ -240,6 +257,60 @@ func (*HTTP) request(ctx context.Context, rt *goja.Runtime, state *common.State,
 		tags["status"] = strconv.Itoa(resp.Status)
 		tags["proto"] = resp.Proto
 
+		if res.TLS != nil {
+			tlsState := res.TLS
+			switch tlsState.Version {
```

Comment:
Let's make this a helper function, then assign `resp.OCSP.StapledResponse` to a fully initialized object. These switches are real lengthy.